File: C++/2357. Make Array Zero by Subtracting Equal Amounts.cpp
#include <bits/stdc++.h>

using namespace std;

class Solution {
 public:
  int minimumOperations(vector<int>& nums) {
    set<int> st;

    for (auto x : nums) {
      if (x != 0) {
        st.insert(x);
      }
    }

    int ans = st.size();
    return ans;
  }
};

File: JavaScript/2357. Make Array Zero by Subtracting Equal Amounts.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var minimumOperations = function (nums) {
  let st = new Set();
  for (let i = 0; i < nums.length; i++) {
    if (nums[i] !== 0) st.add(nums[i]);
  }
  return st.size;
};
